recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - qcelemental <=0.28
     - qcportal >=0.54
     - openff-toolkit-base >=0.12
-    - openff-units <0.3.0
+    - openff-units 
```

Comment:
Small preference for something like `openff-units >0.2.0,<0.4.0` to prevent any breaking changes that may come from 0.4.0, mirroring what happened by necessity in 0.3.0